assets/utils.js

```diff
@@ -17,6 +18,10 @@ import {
     TO_BE_CONFIRMED_TEXT
 } from './agenda/utils';
 
+// Add locales to moment.js
+moment.locale('fr_CA');
+moment.locale('en');
```

Comment:
shouldn't it be based on the config rather than hardcoding it here?

Reply:
I tried to load the config from `window` object here but it's not available yet

Reply:
ok

Reply:
also if we add another locale, we would still need to add it manually [here](https://github.com/superdesk/newsroom/pull/1036/files#diff-c6a35b21c13bfe61f8ac39d33b8d771dR12) cause that's how moment works 🤷‍♂

Reply:
would be good to leave a comment regarding this imo